spec/02_find_length_strings_in_array_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Specification for RubyMonk Primer Problem 2 Find the length of strings in an array
+require_relative '../02_find_length_strings_in_array'
+
+RSpec.describe FindLengthStringsInArray do
+  subject(:strings_in_array) { described_class.new }
```

Comment:
You get a ⭐ for using this as the `subject` instead of `described_class.new.length_finder` (which you probably know is a huge pet peeve of mine)! :)

Reply:
Yeah, subjects should be nouns 😉 .